pkgs/development/compilers/odin/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib
 , fetchFromGitHub
-, llvmPackages_13
+, llvmPackages_18
```

Comment:
```suggestion
, llvmPackages_18
```
This is not the right way of overriding the version of a package.
This should be: `llvmPackages` or else the interface is going to change for overrides.

Reply:
llvmPackages is still at version 17 though, while this PR wants to change it to version 18. do you mean we should specify the version inside `top-level/all-packages.nix`?

Reply:
Or you could use `?`.

Reply:
Yes, specify the version in `callPackage`.

Reply:
Example from another package:
> pkgs/top-level/all-packages.nix
![image](https://github.com/NixOS/nixpkgs/assets/5861043/740daf61-4495-4654-a7a6-518eef6bbc13)